src/sqlast/mod.rs

```diff
@@ -608,3 +614,28 @@ impl FromStr for FileFormat {
         }
     }
 }
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct SQLDrop {
+    pub if_exists: bool,
+    pub names: Vec<SQLObjectName>,
+    pub cascade: bool,
+    pub restrict: bool,
+}
+
+impl SQLDrop {
+    fn to_string_internal(&self, object_type: &str) -> String {
+        format!(
+            "DROP {}{} {}{}{}",
+            object_type,
+            if self.if_exists { " IF EXISTS" } else { "" },
+            self.names
```

Comment:
`comma_separated_string(&self.names),` seems to work here.

Reply:
Good point. Done.